libmscore/measure.cpp

```diff
@@ -1002,7 +1002,8 @@ void Measure::remove(Element* e)
                         // st currently points to an list element that is about to be removed
                         // make a copy now to use on undo/redo
                         StaffType* st = new StaffType(*stc->staffType());
-                        staff->removeStaffType(tick());
+                        if (!tick().isZero())
+                              staff->removeStaffType(tick());
```

Comment:
Does that also remove a staff type change at the very beginning of a score?

Reply:
The StaffTypeChange element is removed. The point is to not remove the Staff's initial StaffType.

Reply:
Outch, of course...